assets/airports/kbos.json

```diff
@@ -398,316 +528,8175 @@
-                "NELIE": ["NELIE"],
                 "GLYDE": ["GLYDE"],
-                "HANAA": ["HANAA"]
+                "HYLND": ["HYLND"],
+                "LBSTA": ["LBSTA"],
+                "LFV": ["LFV"],
+                "LUCOS": ["LUCOS"],
+                "MHT": ["MHT"],
+                "NELIE": ["NELIE"],
+                "ONEPS": ["ONEPS"],
+                "PATSS": ["PATSS"],
+                "PSM": ["PSM"],
+                "REVSS": ["REVSS"],
+                "SEY": ["SEY"],
+                "SPENO": ["SPENO"],
+                "SSOXS": ["SSOXS"]
 
             },
             "draw": [
-                ["BOS", "MHT*"],
-                ["BOS", "ENE*"],
-                ["BOS", "PSM*"],
-                ["BOS", "LFV*"],
-                ["BOS", "ACK*"],
-                ["BOS", "MVY*"],
-                ["BOS", "PVD*"],
-                ["BOS", "SEY*"],
-                ["BOS", "CMK*"],
-                ["BOS", "BAF*"],
-                ["BOS", "CTR*"],
-                ["BOS", "CAM*"],
-                ["BOS", "FRILL*"],
-                ["BOS", "FREDO*"],
-                ["BOS", "DUNKK*"],
-                ["BOS", "BURDY*"],
-                ["BOS", "BOSOX*"],
-                ["BOS", "NELIE*"],
-                ["BOS", "GLYDE*"],
-                ["BOS", "HANAA*"]
+                ["_BOS261002", "WYLYY", "CYOTI"],
+                ["ACK*"], ["BAF*"], ["BLZZR*"],
+                ["BOSOX*"], ["BRUWN*"], ["BURDY*"],
+                ["CELTK*"], ["CTR*"], ["FRILL*"],
+                ["GLYDE*"], ["HYLND*"], ["LBSTA*"],
+                ["LFV*"], ["LUCOS*"], ["MHT*"],
+                ["NELIE*"], ["ONEPS*"], ["PATSS*"],
+                ["PSM*"], ["REVSS*"], ["SEY*"],
+                ["SPENO*"], ["SSOXS*"]
             ]
         }
     },
     "stars": {
+        "GDM4": {
+            "icao": "GDM4",
+            "name": "Gardner Four",
+            "entryPoints": {
+                "ALB": ["ALB", "WIKID"]
+            },
+            "body": ["GDM", "BRONC", "LOBBY", "REVER", "BOS"],
+            "rwy": {
+                "4L": [],
+                "4R": [],
+                "15R": [],
+                "22L": [],
+                "22R": [],
+                "27": [],
+                "32": [],
+                "33L": []
+            },
+            "draw": [
+                ["ALB", "WIKID", "GDM", "BRONC", "LOBBY", "REVER", "BOS"]
+            ]
+        },
         "JFUND1": {
             "icao": "JFUND1",
             "name": "Jayfund One",
             "entryPoints": {
-                "PONCT": ["PONCT", "STOLI", ["MNSTA", "A250"], ["SEETS", "A210|S290"], ["DUBLE", "A180"], ["BASEZ", "A150"]],
-                "WHATE": ["WHATE", ["ONDEC", "A110+"]],
-                "ONDEC": [["ONDEC", "A110+"]]
+                "PONCT": ["PONCT", ["STOLI", "A320-|S350"], ["MNSTA", "A250|S300"], ["SEETS", "A210|S280"], ["DUBLE", "A170"], ["BASEZ", "A140"]]
+            },
+            "body": [["JFUND", "A140"]],
```

Comment:
I understand, and though this _does work_, I would rather take the time to actually fix the problem by adding the feature instead of having little workarounds like this sprinkled throughout. I'll move #32 to the top of the queue in zube.